Source/ACE/Network/GameAction/Actions/GameActionChangeCombatMode.cs

```diff
@@ -9,7 +10,7 @@ public static class GameActionChangeCombatMode
         public static void Handle(ClientMessage message, Session session)
         {
             var newCombatMode = message.Payload.ReadUInt32();
-            session.Player.SetCombatMode((CombatMode)newCombatMode);
+            session.Player.SetCombatMode((CombatMode)newCombatMode, session.Player);
```

Comment:
probably just a lack of understanding on my part.   I am not sure, with all of these issues - should I just close this PR?